cns/kubecontroller/nodenetworkconfig/conversion_windows.go

```diff
@@ -32,6 +33,37 @@ func createNCRequestFromStaticNCHelper(nc v1alpha.NetworkContainer, primaryIPPre
 			NCVersion: int(nc.Version),
 		}
 	}
+
+	if nc.Type == v1alpha.VNETBlock {
```

Comment:
Let's rework the logic to

if DefaultGw == "" && Overlay {
    // set 0th IP to GW
} else if DefaultGw == "" && VnetBlock {
    // set 0th IP as Primary
    // set 1st IP as GW
} else if Vnet Block {
    // set 0th IP as Primary
}

// block to add all remaining IPs from primary block for assignment

// Add all IPs from secondary block for assignment

Reply:
updated